java/README.md

```diff
@@ -1,12 +1,14 @@
 # Fury Java 
+
 ## Code format
+
 ```bash
-mvn -T10 license:format
```

Comment:
If the users want to format code locally, do they need to build hawkeye?
```
# Build
./mvnw clean install -DskipTests
# Run
./hawkeye-cli/target/hawkeye.jar -h
```

Reply:
No. If you read the README of http://github.com/korandoru/hawkeye, the tool is platform agnostic and JDK free. That is, you can run with `docker` - `docker run --rm -v $(pwd):/github/workspace ghcr.io/korandoru/hawkeye-native:v3 format` whose image is only 28MB.

Reply:
I see, users need install docker, and mount current directory into container for format. Can we add this to our doc if users want format automatically.

Reply:
OK.